cmd/config/show_test.go

```diff
@@ -189,7 +189,7 @@ func TestConfigShow(t *testing.T) {
 
 		assert.Contains(t,
 			stderr,
-			`unexpected unclosed action in command`,
+			`Error:  template: config:1: unclosed action`,
```

Comment:
I don't fully understand why this one diverged so much, so maybe someone with Go experience could advise.

Reply:
Maybe just change the Contains to check for "unclosed action" as that seems to be common between the two versions?

Reply:
updated!